src/main/java/org/jetbrains/bsp/bazel/server/bsp/resolvers/actiongraph/ActionGraphV1Parser.java

```diff
@@ -67,7 +49,14 @@ private String getTargetId(String needle) {
         .collect(Collectors.toList());
   }
 
-  private List<AnalysisProtos.Artifact> expandDepsetToArtifacts(Queue<String> idsToExpand) {
+  private List<AnalysisProtos.Artifact> expandDepsetToArtifacts(String idToExpand) {
+    Queue<String> idsToExpand =
```

Comment:
I'm not sure what is happening here, I know that this is old code, but it might be updated a bit

Reply:
I've tried to streamify it a bit, let me know if there's anything else you think should be done!